GUI/AllegroInput.cpp

```diff
@@ -307,28 +357,6 @@ void AllegroInput::Update(void)
         mouse_x = MIN((g_FrameMan.GetResX() * mouseDenominator) - 3, mouse_x);
         mouse_y = MIN((g_FrameMan.GetResY() * mouseDenominator) - 3, mouse_y);
 
-        // Button states/presses, Primary - ACTUALLY make either button work, we don't have use for secondary in menus
-        if (g_UInputMan.MenuButtonHeld(UInputMan::MENU_EITHER))
```

Comment:
It seems this was kind of pointless semi-duplicate code? I'd like to confirm though, what did it do/why did you get rid of it?

Reply:
as it was before, clicks were registered earlier in the function in the normal way,
and then at this point in the function they would be re-registered, and sometimes altered. for example changing a mouse event from "pushed" to "released". i don't know how or why, but it makes no sense.